tests/forte/utils/payload_factory_test.py

```diff
@@ -0,0 +1,142 @@
+# Copyright 2022 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit test for payload factory.
+"""
+import os
+import unittest
+from ft.onto.payload_ontology import (
+    JpegPayload,
+    SoundFilePayload
+)
+from forte.data.data_pack import DataPack
+from forte.utils.payload_factory import register
+
+@register
+class OnlineJpegPayload(JpegPayload):
+    def loading_fn(self):
+        """
+        A function that parses payload meta data and prepare and returns a loading function.
+        
+        This function is not stored in data store but will be used
+        for registering in PayloadFactory.
+        
+        Returns:
+            a function that reads image data from an url.
+        """
+        try:
```

Comment:
Do we assume users to write the import checking part?

Reply:
Maybe not. I am thinking maybe we can assume users should install and import the package properly.